src/test/java/StringAddCalculatorTest.java

```diff
@@ -0,0 +1,40 @@
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import org.junit.jupiter.api.Test;
+
+public class StringAddCalculatorTest {
+
+    @Test
+    public void splitAndSum_one() throws Exception{
+        int result = StringAddCalculator.splitAndSum("1");
+        assertThat(result).isEqualTo(1);
```

Comment:
```suggestion
        int result = StringAddCalculator.splitAndSum(null);
        assertThat(result).isEqualTo(0);
```

TestCase 소스 코드와 내용이 조금 달라요. 😅